genkit-tools/common/src/server/server.ts

```diff
@@ -79,6 +79,33 @@ export async function startServer(manager: RuntimeManager, port: number) {
     res.end();
   });
 
+  app.get('/api/latest/runtime', async (_, res) => {
+    res.writeHead(200, {
+      'Access-Control-Allow-Origin': '*',
+      'Cache-Control': 'no-cache',
+      'Content-Type': 'text/event-stream',
+      Connection: 'keep-alive',
+    });
+
+    // On connection, immediately send the "current" runtime (i.e. most recent)
+    const runtimeInfo = JSON.stringify(manager.getMostRecentRuntime());
+    res.write(`data: ${runtimeInfo}\n\n`);
```

Comment:
Is this an easily parseable/valid JSON response? Is `data` a field in an object here?

Reply:
This is the spec: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#fields

It's old school - key: value, one per line.